controllers/jobs.js

```diff
@@ -0,0 +1,36 @@
+const Job = require('../models/Job');
+const { StatusCodes } = require('http-status-codes');
+const { BadRequestError, NotFoundError } = require('../errors');
+
+
+const getAllJobs = async (req, res) => {
+    res.send('get all jobs');
+}
+
+const getJob = async (req, res) => {
+    res.send('get job');
+}
+
+const createJob = async (req, res) => {
+    req.body.createdBy = req.user.userId;
+    const job = await Job.create(req.body);
+    res.status(StatusCodes.CREATED).json({ job });
+}   
+
+const updateJob = async (req, res) => {
+    res.send('update job');
+}
+
+const deleteJob = async (req, res) => {
```

Comment:
you did not implement the method

Reply:
Hi Taiwo, I did not implement the methods in this PR because that logic is covered in the next lesson (week 10).